easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.6-foss-2023a.eb

```diff
@@ -0,0 +1,153 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>; Guillermo AR Piccoli <guillermo.piccoli@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonPackage'
+
+name = 'MAGeCK-VISPR'
+version = '0.5.6'
+
+homepage = 'https://bitbucket.org/liulab/mageck-vispr/'
+description = """MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://bitbucket.org/liulab/%(namelower)s/get']
+sources = ['v%(version)s.tar.gz']
+checksums = ['b6bdd59d34f9bbb548c88de8948e2aacf7e461a62a74c1b8ef8a73b9b64b2e92']
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Meson', '1.2.3', '', ('GCCcore', '13.2.0'))
```

Comment:
wrong toolchain, don't specify the toolchain at all, and Meson should just be a builddep

Reply:
This requires Meson >= 1.2.1 but it is not available in GCC 12.3.0, should I commit a new Meson eb for 12.3.0 or should I just switch MAGeCK-VISPR to foss 2023b?

Reply:
You can switch to 2023b if you want.

Adding multiple versions of things that are builddependencies is completely OK; they are only used during the build and won't cause conflicts afterwards (which is the challenge in case of things used as normal deps). So your other option is to make a new `Meson-1.4.0-GCCcore-12.3.0.eb`

Reply:
Thank you @Micket I pr Meson-1.4.0-GCCcore-12.3.0.eb as per your advice, because some functions in MAGeCK-VISPR are obsolete in Python 3.11.5 so it is incompatible with the available Meson 1.2 in EB.

Reply:
Really? Something was deprecated in 3.11.5 from 3.11.3 which we are using here? Surely not as I would hope python follows semantic versioning. It's fine to keep this in 12.3.0 if you prefer this toolchain, but I don't see how Python makes any difference here.